shared/studio/components/headerNav/elements.tsx

```diff
@@ -153,6 +154,7 @@ export function HeaderNavCol<LinkProps>({
                     />
                   ) : null}
                   <span>{item.label}</span>
+                  {item.selected && <CheckIcon />}
```

Comment:
I think the check icon is meant to indicate the currently active item, not the one that's being hovered/selected.

Reply:
I'll remove it completely for now, it's a bit tricky to implement it to show currently active item.